app/code/Vaimo/QtyManager/view/frontend/web/js/qty-manager.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright © Vaimo, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+'jquery',
+'underscore',
+'mage/template',
+'text!Vaimo_QtyManager/template/qty-template.html',
+'jquery/ui',
+'mage/validation'
+   ], function ($, _, template, qtyTemplate) {
+    'use strict';
+
+    $.widget('mage.qtyManager', {
+        _create: function () {
+            this._constructorBlock();
+            this._minQtyChecker();
+            this._bind();
+        },
+
+        /**
+         * @description Method for adding increase/decrease buttons template and variable declaration.
+         * And replace quantity input to this template.
+         */
+        _constructorBlock: function () {
+            this.element.after($(template(qtyTemplate, '')));
+            this.block = this.element.next('[data-qty-wrapper]');
```

Comment:
Please move all selectors to option for purpose of widget extendability and consistency in widget methods.